spikeinterface/sorters/pykilosort/pykilosort.py

```diff
@@ -176,6 +176,15 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         ks_probe.xc = locations[:, 0]
         ks_probe.yc = locations[:, 1]
         ks_probe.shank = None
+        if recording.get_channel_gains() is not None:
+            gains = recording.get_channel_gains()
+            if len(np.unique(gains)) == 1:
+                ks_probe.sample2volt = gains[0] * 1e-6
+            else:
+                UserWarning('Multiple gains detected for different channels. Median gain will be used')
```

Comment:
```suggestion
                warnings.warn('Multiple gains detected for different channels. Median gain will be used')
```
And we need to add `import warnings` at the top